### ISSUE TITLE
How do I invite my team to a overv.io project?

### ISSUE THREAD

**Issue by ehausen** [mirumee/overv.io-beta#518]
I have created a overv.io project with a repo and **I want to add another github & overv.io user to the same project.**

But it seems that when I tested with another user I had to create a new overv.io project and add the repo to that and create all the columns again.

Are the projects really per user? 
Does every member of the team have to create their own workspace on overv.io?

Details of my procedure:
- I have created a Github organisation (and enabled overv.io to access it from application settings)
- I have successfully created a new project on _overv.io_ with my org-name/my-test-repo with my real github username
- I have added members to my Github organizaton
- Added test-member to a test-team
- Gave the test-team access to org-name/my-test-repo
- Successfully logged in with _test-member to_ _overv.io_ with Github in another browser
- Was able to create a project with org-name/my-test-repo

**Comment by mirekm** [comment]
Hi Erik,
Overv.io projects are visible to all users being GitHub collaborators of the repo(s) included in this projects. Each collaborator can only see issues from the repo(s) he has access to on GitHub. It's safe to keep a single project containing multiple repos with exclusive lists of collaborators. Then all the settings (like columns) are shared and everyone only sees issues from repos available to him on GitHub.

The only thing required by the other team members (as long as they are already collaborators in your repo(s) on GitHub) is to log in to Overv.io. Then all the Overv.io projects containing any repos they have access to on GitHub (and earlier added to Overv.io) should be available to them.

TLDR;
The only known problem at the moment is that brand new collaborators might not be synced with Overvio on some point yet. Then you might need to refresh your project in Overv.io (or resync it from 'Tracked repositories' tab available on the bottom of the sidebar list of workspaces) and wait for a little bit. We're looking into ways of making this process straightforward and there should be some changes regarding the topic available online soon.

**Comment by ehausen** [comment]
Can you clarify:
"The only known problem at the moment is that brand new collaborators might not be synced with Overvio on some point yet."

I actually managed to access the project with my test-member by going to the URL directly. But I could swear there was no projects already listed when test-member logged in to overv.io? Is that you meant by above.

PS. I really appreciate your fast replies! Remember to enjoy the summer also ;)

**Comment by mirekm** [comment]
If you add new collaborator to your GitHub repo we don't receive any instant notification about it — at the moment it's necessary to pull the new list of collaborators for the repos involved.
This behavior will slightly change soon and we're testing first changes addressing the problem but for now the best way is to send a link to overv.io project if any of your collaborators can't find it on the list.

Thanks! Your feedback is also highly appreciated!

![](http://i.giphy.com/lExXm1vSE7zb2.gif])

**Comment by ehausen** [comment]
Suggestion:
Add help text about that you just said for example here:
<img width="288" alt="screen shot 2015-07-14 at 20 37 50" src="https://cloud.githubusercontent.com/assets/6204129/8681535/49341738-2a68-11e5-9dbb-8bff0cd96a36.png">

**Comment by ehausen** [comment]
By the way...
You stated that

> > Then all the settings (like columns) are shared and everyone only sees issues from repos available to him on GitHub.

I tested it and it's seems to be only columns. Other settings like "Distinguish 'My issues'" and collapsed columns seem to be personal, right?

**Comment by mirekm** [comment]
Yeah, I should have mentioned that: there are shared non-private/general project's settings and personal ones like "Distinguish my issues", collapsed columns or folded milestones.

**Comment by mirekm** [comment]
We've also added a _Refresh button_ to /dashboard view. Now in case you can't find your project (containing repo(s) that you already have access to on GitHub) it will update your list of GitHub repositories and in turn will also reveal all the projects in Overv.io containing these repositories.
